hasher-matcher-actioner/src/OpenMediaMatch/storage/interface.py

```diff
@@ -409,8 +410,19 @@ def bank_delete(self, name: str) -> None:
 
     # Bank content
     @abc.abstractmethod
-    def bank_content_get(self, id: t.Iterable[int]) -> t.Sequence[BankContentConfig]:
-        """Get the content config for a bank"""
+    def bank_content_get(
+        self, id: t.Iterable[int], signal_type: t.Optional[str] = None
+    ) -> t.Sequence[BankContentConfig]:
```

Comment:
unsure: As an alternative to including it in the bank content config, another solution would be to return this as `Mapping[int, Ret]` where ret could include the config and the requested signals separately.